common.mk

```diff
@@ -7,11 +7,19 @@ SHELL=/bin/bash
 # extra make variables/rules from subprojects
 #
 # EXTRA_GENERATOR_REQS - requirements needed for the main generator
-# EXTRA_SIM_FLAGS - runtime simulation flags
-# EXTRA_SIM_CC_FLAGS - cc flags for simulators
-# EXTRA_SIM_SOURCES - simulation sources needed for simulator
-# EXTRA_SIM_REQS - requirements to build the simulator
-#########################################################################################
+# EXTRA_SIM_FLAGS      - runtime simulation flags
+# EXTRA_SIM_CFLAGS     - CFLAGS for building simulators
+# EXTRA_SIM_CXXFLAGS   - CXXFLAGS for building simulators
+# EXTRA_SIM_LDFLAGS    - LDFLAGS for building simulators
+# EXTRA_SIM_SOURCES    - simulation sources needed for simulator
+# EXTRA_SIM_REQS			 - requirements to build the simulator
```

Comment:
A couple of comments here:
- Here you remove the `EXTRA_SIM_REQS`, but that is needed so that extra things can be built before the similar (see `dromajo.mk`). Unintentional delete?
- Nit: Spacing in the comment is off
- Some of these flags aren't used in the VCS `Makefile`. IMO all these variables should be present/used in both files so that running a Verilator vs VCS sim. is similar.

Reply:
agreed about the eventual consistency of VCS/verilator. i just dont' want to deal with VCS this commit

Reply:
i will make a new pr for vcs after this one